src/pages/send-tokens/hooks/use-send-form-validation.ts

```diff
@@ -43,6 +43,11 @@ export const useSendFormValidation = ({ setAssetError }: UseSendFormValidationAr
             errors.amount = `Insufficient balance. Your available balance is ${availableBalance.toString()} STX`;
           }
         }
+        const assetBalance = new BigNumber(selectedAsset.balance);
```

Comment:
This looks good, but I have a question -- we use both BN and BigNumber -- can we just use one? I know BN is a hard requirement for stacks.js.

Reply:
I would love to only use 1, and it should be `BigNumber`, not `bn.js`

Reply:
https://paper.dropbox.com/doc/Proposal-Remove-dependency-on-BigNum-QLbA1U7nvhvqGrQIDR5hD

Reply:
I'm all for it, big plus one from me. @andresgalante any ideas on how best to work with splat in opening up a discussion around this? I do think it would be a big breaking change

Reply:
Let's start a discussion in stacks.js repo, and I'll ask @agraebe to add it to the splat team meeting agenda.

Reply:
☝🏼 added to the agenda. you can start a new discussion on the repo [here](https://github.com/blockstack/stacks.js/discussions/new)